core/src/main/java/org/apache/iceberg/LocationProviders.java

```diff
@@ -160,7 +161,7 @@ private static String pathContext(String tableLocation) {
     }
   }
 
-  private static String stripTrailingSlash(String path) {
+  public static String stripTrailingSlash(String path) {
```

Comment:
Is there a better place for this rather than making it public from `LocationProviders`? We could move it to `CatalogUtil` if you think that's the right place. Or maybe another util class?

Reply:
+1, was thinking, how about if we create a new util class let's say `LocationUtils` under iceberg.utils package, more such helper functions if required could be placed here.